nautobot/core/testing/mixins.py

```diff
@@ -62,6 +62,13 @@ def setUpNautobot(self, client=True, populate_status=False):
             # Force login explicitly with the first-available backend
             self.client.force_login(self.user)
 
+    def tearDown(self):
+        """Clear lru_cache data to avoid leakage of information between test cases when running in parallel."""
```

Comment:
Are these the only models that use LRU cache?

Reply:
AFAIK yes. There's a few other uses of lru_cache in some authentication related code but those don't seem to be impacting tests.